app/src/main/java/one/mixin/android/job/StorageCleanJob.kt

```diff
@@ -0,0 +1,47 @@
+package one.mixin.android.job
+
+import com.birbit.android.jobqueue.Params
+import kotlinx.coroutines.runBlocking
+import one.mixin.android.MixinApplication
+import one.mixin.android.R
+import one.mixin.android.extension.fileSize
+import one.mixin.android.extension.getFileNameNoEx
+import one.mixin.android.extension.getMediaPath
+import one.mixin.android.extension.isUUID
+import one.mixin.android.util.clear.CleanNotification
+import timber.log.Timber
+
+class StorageCleanJob :
+    BaseJob(Params(PRIORITY_UI_HIGH).groupBy(GROUP_ID).requireNetwork().persist()) {
+
+    companion object {
+        private const val serialVersionUID = 1L
+        const val GROUP_ID = "convert_data_group"
```

Comment:
- priority can be lower
- network required is not necessary
- need another `GROUP_ID`